libs/knowledge-store/ragstack_knowledge_store/links.py

```diff
@@ -35,20 +35,20 @@ def __init__(self, kind: str, tag: str) -> None:
         super().__init__(kind=kind, tag=tag, direction="bidir")
 
 
-LINK_TAGS = "link_tags"
+LINK_SET = "link_set"
 
 
-def get_link_tags(doc_or_md: Dict[str, Any]) -> Set[LinkTag]:
-    """Get the link-tag set from a document or metadata.
+def get_link_set(doc_or_md: Dict[str, Any]) -> Set[Link]:
```

Comment:
Maybe just `get_links` ?

Reply:
`get_links` seems more approachable - the fact that it's a set probably isn't that important.